config/locales/en.yml

```diff
@@ -487,6 +490,49 @@ en:
         reject: 'rejected'
         suspend: 'suspended'
         resume: 'resumed'
+      bulk:
```

Comment:
This whole thing could be replaced with a single block:

```
    bulk:
      send_emails:
        success:
          one: The email was sent successfully.
          other: Successfully sent %{count} emails.
      change_states:
        success:
          one: The %{model}'s state was changed successfully.
          other: Successfully changed the state of %{count} %{model}s.
      change_plans:
        success:
          one: The %{model}'s plan was changed successfully.
          other: Successfully changed the plan of %{count} %{model}s.
```

But it would be a bit cumbersome to transform the current context to the right `model`.
All data is within `controller_path` (e.g. "buyers/service_contracts/bulk/change_states"), but yeah, probably it's cleaner this way.